internal/kubebuilder/cmdutil/cmdutil.go

```diff
@@ -16,25 +16,31 @@ limitations under the License.
 
 package cmdutil
```

Comment:
Can this entire package not be removed?

Reply:
It will be removed with plugin phase 1.5 so it didnt make sense to expose it now and then remove it inmediately after